### DISCUSSION TITLE
Reference

### DISCUSSION THREAD

**Discussion by Anindoroyapu** [Bipon-Dev/kpi-alumni-frontend#23] [General]
Reference

**Comment by Anindoroyapu** [comment]
https://alumni.iitd.ac.in/

**Comment by bipon68** [comment]
Dummy API
https://jsonplaceholder.typicode.com/

**Reply by Anindoroyapu** [reply]
ok

**Comment by Anindoroyapu** [comment]
https://alumni.duetbd.org/

DUET alumni .. Everyone should visit this. I think it will be useful for us. They have the same idea as us.

**Comment by bipon68** [comment]
AI Tools
- https://bolt.new/
- https://replit.com/pricing
- https://claude.ai/

**Comment by panthosarkar** [comment]
https://app.tealhq.com/ 

This Is an  auto  CV or Resume Builder website

**Comment by bipon68** [comment]
UI Concept 
http://www.themestarz.net/html/universo/event-listing.html

**Comment by Anindoroyapu** [comment]
For message option 


https://www.tawk.to/?utm_source=tawk-messenger&utm_medium=link&utm_campaign=referral&utm_term=59b62b9a4854b82732fef419

**Comment by bipon68** [comment]
Load Testing tool
https://k6.io/

**Comment by bipon68** [comment]
Re:tune is a platform that enables anyone to build, customize, and publish AI solutions
https://retune.so/

**Comment by bipon68** [comment]
DevOps
https://youtube.com/playlist?list=PLdpzxOOAlwvKwTyYNJCUwGPvql0TrsPgv&si=BFpQOjdVaOeksLiL

**Comment by bipon68** [comment]
A developer-first solution for custom communication experiences. Securely integrate with every email, calendar, and contacts provider using a single interface
https://www.aurinko.io/
Nylas

**Comment by bipon68** [comment]
**Changelog**
https://changelog.md/kpialumni

**Comment by bipon68** [comment]
সিম্পল সার্ভার বা প্ল্যাটফর্ম (যেমন Render, Railway, Vercel) যথেষ্ট। Appocean reference
https://vercel.com/
https://railway.com/
https://render.com/
